shim/src/intrinsics.js

```diff
@@ -285,7 +281,7 @@ export function getIntrinsics(sandbox) {
         // %unescape%
         unescape: global.unescape,
 
-        // TODO: other special cases
+        // TODOther special cases
```

Comment:
Was this caused by some automatic text text replacement?

Reply:
Thanks. Fixed. Appreciated!